packages/context/src/inject.ts

```diff
@@ -331,21 +395,28 @@ export function describeInjectedArguments(
   return meta || [];
 }
 
-function resolveByTag(
+function resolveBindings(
```

Comment:
Similarly here, I don't see how is this change relevant to binding configuration. Can you revert it please?